### PR TITLE
Proposal for initial implementation

### PR DESCRIPTION
**Pull request by redmaner**
This is my initial proposal for a Nimiq RPC client library in Go. 

**Development and review**
The development history can be found in the [original repository](https://github.com/redmaner/go-nimiq-rpc/). @terorie has already provided some feedback that has been incorporated in this proposal. 

**About docs**
This library does not provide a separate /docs folder, because Go automatically generates it on godoc. See for example: https://godoc.org/github.com/redmaner/go-client. The documentation for github.com/nimiq-community/go-client will be generated once this pull request is merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,63 @@
...
+[![GoDoc](https://godoc.org/github.com/nimiq-community/go-client?status.svg)](https://godoc.org/github.com/nimiq-community/go-client)
+
 
 > Go implementation of the Nimiq RPC client specs.
 
-## Usage
+## About
+A Nimiq RPC client library in Go. This client library implements the [Nimiq RPC specification](https://github.com/nimiq/core-js/wiki/JSON-RPC-API#remotejs-client). This client uses the jsonrpc library to handle JSON-RPC 2.0 requests and responses. For more information about this library see the [jsonrpc documentation](https://godoc.org/github.com/ybbus/jsonrpc)
+
+### What is Nimiq?
```
Comment (riptl) [inline README.md]: I think we can remove the "What is Nimiq" section, people using this library should already know :-)

File: README.md [hunk 2]
```diff
@@ -1,26 +1,63 @@
...
+A blockchain technology inspired by Bitcoin but designed to run in your browser. It is money by nature but capable to do much more.
+See [the Nimiq website](https://nimiq.com) for more information.
+
+### Usage
+This library is fully Go module compatible. See the full documentation on [GoDoc](https://godoc.org/github.com/nimiq-community/go-Client)
+An example of using this library can be found below:
+
+```
+package main
```
Comment (riptl) [inline README.md]: For the sake of simplicity, let's just do three lines for the example

```go
client := nimiqrpc.NewClient("address.to.nimiqnode.com")
balance, _ := client.GetBalance("NQ52 V4BF 52J3 0PM6 BG4M 9QY1 RUYS UAL6 CJD2")
fmt.Println("Balance: ", balance)
```

**Review by riptl — CHANGES_REQUESTED** [review]
Thanks a lot for your work, this is looking great!
I just have two documentation suggestions, otherwise let's get this merged.

File: README.md (hunk 2 above)
Comment (redmaner) [reply README.md]: I can live with that

File: README.md (hunk 1 above)
Comment (redmaner) [reply README.md]: True. Added it for completeness, but is probably redundant information for someone who wants to use the library.

**Comment by redmaner** [comment]
Simplified README according to your feedback

**Comment by riptl** [comment]
Thanks again, ready for merge! @svub

**Comment by redmaner** [comment]
Epic. Thank you too for reviewing.

**Pull request merged**